src/plugins/unified_field_list/public/components/field_stats/field_stats.tsx

```diff
@@ -76,6 +76,7 @@ export interface FieldStatsProps {
   overrideMissingContent?: (params: {
     element: JSX.Element;
     noDataFound?: boolean;
+    supported?: boolean;
```

Comment:
We could also merge `noDataFound?: boolean;` and `supported?: boolean;` into a single `reason?: 'no-data' | 'unsupported'` prop.